app/src/organisms/ProtocolSetup/RunSetupCard/index.tsx

```diff
@@ -33,15 +44,11 @@ export function RunSetupCard(): JSX.Element | null {
     standardDeckDef as any
   )
   const robot = useSelector((state: State) => getConnectedRobot(state))
-
-  const ROBOT_CALIBRATION_STEP_KEY = 'robot_calibration_step' as const
-  const LABWARE_SETUP_KEY = 'labware_setup_step' as const
-  const MODULE_SETUP_KEY = 'module_setup_step' as const
-
-  type StepKey =
-    | typeof ROBOT_CALIBRATION_STEP_KEY
-    | typeof MODULE_SETUP_KEY
-    | typeof LABWARE_SETUP_KEY
+  const robotName =
+    robot?.name !== null && robot?.name !== undefined ? robot?.name : ''
```

Comment:
```suggestion
    robot?.name != null ? robot?.name : ''
```

This should be identical.  `!= null` is equal to `!== null | undefined`